src/app/pages/news/9.4/08-tracing-system-overview.md

```diff
@@ -1,7 +1,7 @@
 ## Slow Requests and Traffic Graph {#tracing}
 
 It is key for an orchestration platform like Axon Ivy to provide tools to analyze problems. 
-The new Slow Requests and Traffic Graph views in the Engine Cockpit provide you with all 
+The new "Slow Requests and Traffic Graph" views in the Engine Cockpit provide you with all 
```

Comment:
to different views: "Slow Requests" and "Traffic Graph" views

Reply:
we will go for peters voice

Reply:
I couldn't remove lli thats why :(

Reply:
I changed the graphs to italic, and removed TThe Ttypo we had in another file.